presto-hive/src/main/java/com/facebook/presto/hive/HiveClientConfig.java

```diff
@@ -134,6 +134,8 @@
 
     private boolean tableStatisticsEnabled = true;
 
+    private int partitionStatisticsSampleSize = 100;
```

Comment:
> nit: no need for empty line before this

Since the `tableStatisticsEnabled` and `partitionStatisticsSampleSize` are from the same "domain" i don't think we need one. Going to remove that.

> partitionStatisticsSampleSize

Setting this property to a very high value effectively disables the sampling.